k8s-manifest/advanced/01_db1000n_daemonset.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: db1000n
+  namespace: db1000n
+  labels:
+    app: db1000n
+spec:
+  selector:
+    matchLabels:
+      name: db1000n
+  template:
+    metadata:
+      labels:
+        name: db1000n
+    spec:
+      containers:
+      - name: db1000n
+        image: oneodinscrow/db1000n-light
```

Comment:
Is that correct image?

Reply:
I'll change to the original one, that is my personal repo, where I run test